src/ur5_simulation/ur5model.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 12 20:48:22 2021
+
+@author: Rafael Queiroz
+"""
+
+import pandas as pd
+import numpy as np
+from math import cos, sin
+
+class UR5:
+        
+    def __init__(self, d1=0.089159, a2=-0.42500, a3=-0.39225, d4=0.1091,
+                 d5=0.09465, d6=0.0823):
+        self.d1 = d1
+        self.a2 = a2
+        self.a3 = a3
+        self.d4 = d4
+        self.d5 = d5
+        self.d6 = d6 
```

Comment:
Just for python code best practices, use two underlines before private variables and methods. For this example, it would be `self.__d1`. This helps us, code developers, to know which variable and method is private or not. Apply this rule for the whole code, please!

Reply:
Ok!